src/Controllers/controllerGame.class.php

```diff
@@ -121,10 +119,52 @@ public function initGame(string $code, ?array $settings): void
             $settings = [];
         }
 
+        $baseUrl = $gameSettings["settings"]["serverPort"] != null ? $gameSettings["settings"]["serverAddress"] . ":" . $gameSettings["settings"]["serverPort"] : $gameSettings["settings"]["serverAddress"];
+
+        $data = [
+            "settings" => $settings,
+            "players" => array_map(function ($player) {
+                return [
+                    'uuid' => $player["player"]->getUuid(),
+                    'username' => $player["player"]->getUsername(),
+                    'token' => $player["token"]
+                ];
+            }, $gameRecord->getPlayers()),
+        ];
+
+        $ch = curl_init($baseUrl . "/" . $gameRecord->getCode() . "/init");
+        curl_setopt($ch, CURLOPT_POST, true);
+        curl_setopt($ch, CURLOPT_POSTFIELDS, json_encode($data)); // Envoyer le JSON
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true); // Obtenir la réponse
+        curl_setopt($ch, CURLOPT_HTTPHEADER, [
+            "Content-Type: application/json", // Indiquer que les données sont au format JSON
+        ]);
+
+        // Exécuter la requête
+        $response = curl_exec($ch);
+
+        // Vérifier les erreurs
+        if (curl_errno($ch)) {
+            echo "Erreur cURL : " . curl_error($ch);
+        } else {
+            // Afficher la réponse
+            echo "Réponse : " . $response;
+        }
```

Comment:
Attention, ce format invalide le retour json_encode plus bas car il y aura des échos avant. 

Il vaut mieux faire, s'il y a une erreur, :
```suggestion
        if (curl_errno($ch)) {
            echo json_encode([
                "success"=>false, 
                "message"=>curl_error($ch)]
            );
            exit; ;
        }
```